lib/model/news.dart

```diff
@@ -0,0 +1,40 @@
+class News {
+  final String title;
+  final String content;
+  final String imageURL;
+  final String websiteURL;
+
+  const News({
+    required this.title,
+    required this.content,
+    required this.imageURL,
+    required this.websiteURL,
+  });
+
+  factory News.fromJson(Map<String, dynamic> json) {
+    Map<String, dynamic> yoastContent = json["yoast_head_json"];
+    String title = json["title"]["rendered"];
+    String content = yoastContent["og_description"];
+    String imageURL = yoastContent["og_image"][0]["url"];
+    String websiteURL = yoastContent["og_url"];
+
+    return News(
+      title: title,
+      content: content,
+      imageURL: imageURL,
+      websiteURL: websiteURL,
+    );
+  }
+
+  static List<News> createFakeData() {
+    return List.generate(
+      2,
+      (index) => const News(
+        title: "NewsTitle",
+        content: "NewsContent",
+        imageURL: "NewsImageURL",
```

Comment:
Das News Item wirft mit der FakeData immer eine Exception während dem laden. 
Ich weiß nicht was da eine elegante Lösung wäre, evtl. einfach imageURL nullable und in dem Fall kein Bild anzeigen (oder ein Standardbild)? Da da ja eh der Skeletonizer drüber ist sieht man das Bild ja gar nicht

```
The following ArgumentError was thrown resolving an image codec:
Invalid argument(s): No host specified in URI file:///NewsImageURL
```

Reply:
Sollte bei solchen Exception mal mehr drauf achten. Ist mir garnicht aufgefallen

Reply:
Ich übergebe jetzt einfach einen leeren String bei der Mock Data und hab eine Abfrage hinzugefügt, dass eine farbige Platzhalterbox verwendet wird, wenn ein leerer String kommt.